doc/dev/README.md

```diff
@@ -0,0 +1,6 @@
+# Azure SDK for Rust – Development
+
+This folder collects in-progress developer documentation for contributors building Azure SDK clients, with a focus on management plane generation. Start with the guides below and follow the repository contribution docs for anything not covered here.
+
+-   [Generating Management Plane Crates from TypeSpec](generate.md)
+-   [Creating a TypeSpec-Based Client (Data & Management)](../new-typespec-based-client.md)
```

Comment:
I would change the order of this list since folks should read the `Creating a TypeSpec-Based Client (Data & Management)` doc first. 

I would also look at making sure there isn't any duplication of information across these docs so we don't have to update the info in two places if it changes. 

The other doc should probably also get a link to the Management Specifics docs letting folks know they should read it and go through the steps if they are working on a MGMT plane library. 

Then write the `Generating Management Plane Creates from TypeSpec` doc assuming folks have already read the other doc, and update the title to something like Management Plane Generation Additional steps, or something like that.